lang/CMakeLists.txt

```diff
@@ -297,7 +297,7 @@ endif(ALSA_FOUND)
 
 if (AUDIOAPI STREQUAL portaudio)
     if(WIN32 AND NOT MSYS)
-        target_link_libraries(libsclang portaudio)
+        target_link_libraries(libsclang portaudio_static)
```

Comment:
if, after https://github.com/supercollider/supercollider/pull/4925/files#diff-af3b638bc2a3e6c650974192a53c7291R96,  you `set(PORTAUDIO_LIBRARIES portaudio_static)`, it appears you could entirely eliminate this `if` and the one in server/scsynth/CMakeLists.txt